standard/expressions.md

```diff
@@ -1611,7 +1611,7 @@ A tuple expression has a type if and only if each of its element expressions `Ei
 
 A tuple expression is evaluated by evaluating each of its element expressions in order from left to right.
 
-A tuple value can be obtained from a tuple expression by converting it to a tuple type ([§10.2.13](conversions.md#10213-implicit-tuple-conversions)), by reclassifying it as a value ([§12.2.2](expressions.md#1222-values-of-expressions))) or by making it the target of a deconstructing assignment ([§12.21.2](expressions.md#12212-simple-assignment)).
+A tuple value is obtained from a tuple expression by evaluating it and storing the result in corresponding `System.ValueTuple<...>` type, and initializing each of its fields in order from left to right by evaluating the corresponding tuple element expression of `E`, converting it to the corresponding element type of `T` using the implicit conversion found, and initializing the field with the result.
```

Comment:
System.ValueTuple isn't mentioned anywhere else in this file. Shouldn't we be a bit more specific? There are wrinkles in how tuples map to the ValueTuple types. For instance, a tuple type `(T1, T2, T3, T4, T5, T6, T7, T8)` must be represented as nested ValueTuples, because the TRest type argument of a tuple must always itself be a nested tuple: `ValueTuple<T1, T2, T3, T4, T5, T6, T7, ValueTuple<T8>>`